### ISSUE TITLE
Set-up plumbing to send relevancy scores to api 

### ISSUE THREAD

**Issue by tomy-pg** [pg-irc/QA-Tool#18]
Set-up the plumbing to send relevancy scores to the api. 

Related to [Issue #20](https://github.com/pg-irc/QA-Tool/issues/20)

For each service that is scored the following information must be sent to the database:
user, topic_id, location_id, score, algorithm_id. 

The api will contain the topics, locations and service list algorithms available to test. 
Build plumbing to receive these things as well as set-up the post request that will contain the relevancy score for service. 

**Users are out of scope for this specific issue.** 

### Acceptance Criteria: 

- [x] score buttons (high, low, not relevant) are generated for each service in listing 
- [x] locations, topics and service list algorithms are fetched from the api to be used on the client 

### Steps for QA:
This issue has been developed in parallel with this PR: https://github.com/pg-irc/pathways-backend/pull/468

1. `cd pathways-backend`
2. `git checkout feature/QA_tool_Issue#15`
3. populate two rows in qa_tool_algorithm table with the following data:
```
id: 1
url: https://fierce-ravine-89308.herokuapp.com/
and 
id: 2
url: https://pathways-production.herokuapp.com/

 and populate one row in the  qa_tool_searchlocation table with the following data:
id: 1
name: Vancouver
point: SRID=4326;POINT(-123.1207 49.2827)
```
4. `./manage.py runserver`

5. `cd QA-Tool`
6. `git submodule update --init --recursive` 
7.  create a `.env` file and add the variable `REACT_APP_API_URL='http://127.0.0.1:8000/'`
8. `yarn start`

9. On app start, click on both the topics and locations drop-downs and verify that these dropdowns are populated
10. Choose a topic and location from the drop-downs and click send
11. Verify that a list of 5 services are displayed with score buttons labelled 1, 2 and 3.
